grpc/service_config/service_config.proto

```diff
@@ -187,6 +187,9 @@ message GrpcLbConfig {
   // Multiple LB policies can be specified; clients will iterate through
   // the list in order and stop at the first policy that they support.
   repeated LoadBalancingConfig child_policy = 1;
+  // Optional.  If specified, overrides the name of the target to be sent to
+  // the balancer.
+  string target_name = 2;
```

Comment:
@markdroth and I talked. I understand now that we need such a field, and this isn't a normal "target" string. Let's go with `service_name`, to try to match the nomenclature we're using in the [grpclb protocol](https://github.com/grpc/grpc-proto/blob/2e92786b5ac2675e7315c7a5299b8e29886cb444/grpc/lb/v1/load_balancer.proto#L48).

Reply:
Done.